ajax/libs/youtube-google-analytics/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "youtube-google-analytics",
+  "filename": "lunametrics-youtube.gtm.min.js",
+  "version": "8.1.3",
+  "description": "This is a plug-and-play tracking solution for tracking user interaction with YouTube videos in Google Analytics. It will detect if GTM, Universal Analytics, or Classic Analytics is installed on the page, in that order, and use the first syntax it matches unless configured otherwise. It include support for delivering hits directly to Universal or Classic Google Analytics, or for pushing Data Layer events to be used by Google Tag Manager.",
+  "keywords": [
+    "analytics",
+    "youtube",
+    "tracking"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/lunametrics/youtube-google-analytics.git",
+    "basePath": "",
+    "files": [
+      "lunametrics-youtube.gtm*"
+    ]
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/lunametrics/youtube-google-analytics.git"
+  },
+  "license": {
```

Comment:
Let's use MIT license to let it as same with upstream.
`"license": "MIT",`
https://github.com/lunametrics/youtube-google-analytics/blob/master/package.json#L20